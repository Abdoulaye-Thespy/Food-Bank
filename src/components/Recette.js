import React from 'react';
import PropTypes from 'prop-types';

function Recette({ handleChange }) {
  return (
    <div>
    <h4>Notre recette<h4>
    </div>
  );
}

Recette.propTypes = {
};
export default Recette;
