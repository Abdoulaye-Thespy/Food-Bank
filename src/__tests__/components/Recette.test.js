import React from 'react';
import '../../setupTests';
import shallowWrapper from '../../setupTests';
import Recette from '../../components/Recette';

describe('Recette component', () => {
  
  const recetteTrial = {
    recette: {
      idMeal: '52895',
      strMeal: 'English Breakfast',
      strMealThumb: 'https://www.themealdb.com/images/media/meals/utxryw1511721587.jpg',
    },
    details: jest.fn(),
    id: '1',
  };
    const page = shallowWrapper(Recette, recetteTrial);

  test('should render page', () => {
    expect(page).toBeTruthy();
    expect(page).not.toBeFalsy();
    expect(page).toMatchSnapshot();
  });

  test('should have a link to detail page', () => {
    const LinkToDetailPage = page.find('.btn btn-info btn-details');

    expect(LinkToDetailPage).toHaveLength(1);
    expect(LinkToDetailPage).not.toHaveLength(0);
    expect(LinkToDetailPage.text()).toEqual('Details');
    expect(LinkToDetailPage.text()).not.toEqual('');
  });
});