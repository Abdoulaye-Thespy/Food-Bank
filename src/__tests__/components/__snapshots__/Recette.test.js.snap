// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recette component should render page 1`] = `ShallowWrapper {}`;
